scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -352,21 +353,59 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
         * Compute an id for the block with its content and its position.
         * Used as `track by` value, it allows the blocks to be well rendered.
         */
-        generateBlockId: function(block) {
-            function hashCode(string) {
+        generateBlockId: function(block, index) {
```

Comment:
This function is also called somewhere else, without this `index` parameter. Can it be an issue ?
(You have `vm.getBlockPosition()` if needed)